catalog-info.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: kubernetes-scanner
+  annotations:
+    github.com/project-slug: snyk/kubernetes-scanner
+    github.com/team-slug: snyk/cloud-engines
+spec:
+  type: external-tooling
```

Comment:
What does this mean exactly? This is a tool that Snyk users run in their own infra. In that sense it is similar to the main CLI, if this helps to classify it.